integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -322,6 +322,17 @@ public void testMonth() throws IOException {
     verifyDataRows(result, rows(9));
   }
 
+  @Test
+  public void testMonth_Of_Year() throws IOException {
+    JSONObject result = executeQuery("select month_of_year(date('2020-09-16'))");
```

Comment:
we should have a test for `month_of_year(datetime('...'))`

Reply:
For all declared in definition (`DateTimeFunction.java`) types

Reply:
This function has been implemented as an alias for the `week` function that is already implemented. Testing the `week` function on the CLI, it returns null when using a datetime. I am not sure why, but I will look into it.

Reply:
Nevermind. The problem was I had an incorrect format for datetime. I have fixed the test.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/174/commits/db2fab6fc41c221dc6c976f358874b9e2ac91899